primitiv/c/internal.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2017 The primitiv Authors. All Rights Reserved. */
+#include <config.h>
+
+#include <primitiv/c/internal.h>
+
+namespace primitiv_c {
```

Comment:
`primitiv::c` is better?

Previously, I used `primitiv::python` in Python frontend.
(C++ code was completely converted to Cython now.)

Reply:
I also think so.